multiversx_sdk_cli/projects/project_rust.py

```diff
@@ -53,15 +54,16 @@ def prepare_build_wasm_args(self, args: List[str]):
             self.get_output_folder()
         ])
 
+    def check_if_sc_meta_is_installed(self):
+        which_sc_meta = shutil.which("sc-meta")
```

Comment:
How about checking if the `rust` dependency is installed (thus, all parts of the rust "module")?

Reply:
`check_if_sc_meta_is_installed` can actually be deleted. Before the code reaches this part, the `ensure_dependencies_installed()` is called. That calls the `install()` method of the `Rust` module. If not all the dependencies are installed, it should install them.